.gitignore

```diff
@@ -1,4 +1,3 @@
-rel-eng/releasers.conf
```

Comment:
Previously `releasers.conf` was ignored, as it contained information about dist-git and internal COPR. Tito needs this file, so the releasing engineer had to have this file there.

I removed the copr configuration and now `releasers.conf` contains only a distgit releaser and the only internal info there is branch name